scripts/mkstatic.py

```diff
@@ -59,6 +59,12 @@
                            ),
     }
 
+# Type and variable declarations for arrays of pointers, required for optional/inactive variables
+TMPPTR_ARR_TYPE_DECLARATION = '''type :: {pointer_type_name}
+         {tmpptr_def}
+      end type {pointer_type_name}'''
+TMPPTR_ARR_DECLARATION = '''type({pointer_type_name}), dimension({dims}) :: {localname}_array'''
```

Comment:
Should `{localname}_array` be `{localname}_array => null()`?

Reply:
I wanted to try w/o first (just in case that the resetting would interfere with other threads)